src/components/views/rooms/RoomTile.js

```diff
@@ -208,7 +208,7 @@ module.exports = React.createClass({
             }
 
             const RoomTileContextMenu = sdk.getComponent('context_menus.RoomTileContextMenu');
-            const elementRect = e.target.getBoundingClientRect();
+            const elementRect = this.refs.badge.getBoundingClientRect();
```

Comment:
Hmm, sorry, but I think if we're making it appear on right click, we should probably make appear where the cursor is like a normal context menu.

Reply:
this requires rejigging things because current context menu appears at the right edge of the node you clicked, shall do as you're right its more expected